atmos_model.F90

```diff
@@ -2490,6 +2491,15 @@ subroutine assign_importdata(jdat, rc)
               GFS_data(nb)%Coupling%hsnoin_cpl(ix) = min(hsmax, GFS_data(nb)%Coupling%hsnoin_cpl(ix) &
                              / (GFS_data(nb)%Sfcprop%fice(ix)*GFS_data(nb)%Sfcprop%oceanfrac(ix)))
               GFS_data(nb)%Sfcprop%zorli(ix)       = z0ice
+              tem = GFS_data(nb)%Sfcprop%tisfc(ix) * GFS_data(nb)%Sfcprop%tisfc(ix)
+              tem = con_sbc * tem * tem
+              if (GFS_data(nb)%Coupling%ulwsfcin_cpl(ix) > zero) then
+                GFS_data(nb)%Sfcprop%emis_ice(ix)    = GFS_data(nb)%Coupling%ulwsfcin_cpl(ix) / tem
+                GFS_data(nb)%Sfcprop%emis_ice(ix)    = max(0.9, min(one, GFS_data(nb)%Sfcprop%emis_ice(ix)))
+              else
+                GFS_data(nb)%Sfcprop%emis_ice(ix)    = 0.96
+              endif
+              GFS_data(nb)%Coupling%ulwsfcin_cpl(ix) = tem * GFS_data(nb)%Sfcprop%emis_ice(ix)
```

Comment:
Well, as you can see I am limitng the emissivity to between 0.9 to 1.0.  Because of this I am recomputing.
(by the way I suspect that CICE is sending "zero:" upward longwave - this was responsible for earlier rrtmgp crash.

Reply:
I see, thanks. I am wondering how the previous version computes the ulwsfcin_cpl in rrtmgp test. If I remember correctly, @DeniseWorthen checked the upward long wave from ice when we were debugging the P7 restart issues, there is no issue with this field.

Reply:
Would the zero lwup be on the first timestep (did you code crash immediately)? Or was it on later timesteps?